jaxrs-tck/src/main/java/ee/jakarta/tck/ws/rs/ee/rs/container/requestcontext/JAXRSClientIT.java

```diff
@@ -113,6 +115,7 @@ public void abortWithTest() throws Fault {
    * Throws IOException.
    */
   @Test
+  @Tag("servlet")
   public void getAcceptableLanguagesTest() throws Fault {
     setProperty(Property.REQUEST_HEADERS, "Accpet-Language:en-us");
```

Comment:
Just noticed that there is a typo in "Accpet". I understand that this has nothing to do with this PR, but could somebody confirm that this actually *is* a bug? I wonder why this never had been noticed. Thanks.

Reply:
@alwin-joseph Let's fix that typo since we are at it, but let's not reset the fast-track time.

Reply:
Fixed the typo.